keybase/platform_windows.go

```diff
@@ -125,6 +139,37 @@ func CheckRegistryUninstallCode(productID string, log Log) bool {
 	return false
 }
 
+// Read all the runonce subkeys and find the one with "Keybase" in the name.
+func checkRebootPending(wow64 bool) (result bool) {
```

Comment:
This should return (bool, error) and then have the caller log error?

I personally stay away from named return values in method signature.

`func checkRebootPending(wow64 bool) (bool, error)`